rpc_jobs/rpc_octavia.yml

```diff
@@ -32,39 +32,39 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
-- project:
-    name: "rpc-octavia-pike-pre-merge"
-
-    repo_name: "rpc-octavia"
-    # URL of the rpc-octavia repository
-    repo_url: "https://github.com/rcbops/rpc-octavia"
-
-    series: "pike"
-
-    # currently we only do master and pike. Once RPC-O switches
-    # to Queens we will retire rpc-octavia and use the
-    # upstream installer
-    branches:
-      - "pike.*"
-
-    # Use image for RPC-O install
-    image:
-      - xenial_snapshot:
-          SLAVE_TYPE: "nodepool-rpc-r16-xenial_no_artifacts-swift"
-
-    # Octavia ignores that setting for now
-    scenario:
-      - "functional"
-
-    # Octavia ignores that setting for now
-    action:
-      - "test"
-
-    jira_project_key: "K8S"
-
-    # Link to the standard pre-merge-template
-    jobs:
-      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
+#- project:
```

Comment:
I'm not sure I understand the purpose of commenting these out. It results in noise which has no purpose. If the intent is to restore this one day, then a look run git history provides what it was. I would recommend simply removing it.